api/v1alpha1/openapi.yaml

```diff
@@ -815,6 +819,40 @@ components:
       required:
         - name
 
+    Ipv4Config:
+      type: object
+      properties:
+        ipAddress:
+          type: string
+          format: ipv4
+          x-oapi-codegen-extra-tags:
+            validate: "required,ip_address,max=15"
```

Comment:
there is already a validator for ip https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Internet_Protocol_Address_IPv4